ql/models/shortrate/onefactormodel.hpp

```diff
@@ -103,10 +103,15 @@ namespace QuantLib {
         Real probability(Size i, Size index, Size branch) const {
             return tree_->probability(i, index, branch);
         }
+        void setSpread(double spread)
+        {
+            spread_=spread;
+        }
       private:
         boost::shared_ptr<TrinomialTree> tree_;
         boost::shared_ptr<ShortRateDynamics> dynamics_;
         class Helper;
+        double spread_;
```

Comment:
I'd replace `double` with `Spread` (also above).

Reply:
I'll have a look at this, thanks